snippets/image.liquid

```diff
@@ -81,5 +81,6 @@ xxl - Image is as wide as the whole viewport
   widths: image_width,
   sizes: image_sizes,
   focal: focal,
-  alt: alt
+  alt: alt,
+  keep_type: keep_type
```

Comment:
Should we update the comment at the top of this file to add the description and usage of the `keep_type` parameter?